RequestService/RequestService.Repo/Repository.cs

```diff
@@ -997,5 +1057,45 @@ public async Task<int> GetRequestIDFromGuid(Guid guid)
                 return -1;
             }
         }
+
+        public List<ShiftJob> GetUserShiftJobsByFilter(GetUserShiftJobsByFilterRequest request)
+        {
+            byte requestTypeShift = (byte)RequestType.Shift;
+            var jobs = _context.Job
+                .Include(i => i.NewRequest)
+                .ThenInclude(i => i.Shift)
+                .Where(x => x.VolunteerUserId == request.VolunteerUserId && x.NewRequest.RequestType == requestTypeShift).ToList();
+
+            if (jobs == null || jobs.Count == 0)
+            {
+                return null;
```

Comment:
Are these the same case?
* null feels like an error
* Count == 0 feels like no jobs, so more appropriate to return an empty list?